fanficfare/adapters/adapter_inkbunnynet.py

```diff
@@ -157,10 +157,11 @@ def doExtractChapterUrlsAndMetadata(self, get_cover=True):
             self.story.addToList('genre', stripHTML(kword))
 
         # Getting the Category
+        category = bookdetails.findChildren('div', recursive=False)[2].find('span', string='Type:').parent
+        category.find('span').decompose()
+        self.story.setMetadata('category', stripHTML(category).strip())
         for div in bookdetails.find_all('div'):
-            if 'Details' == stripHTML(div).strip():
-                self.story.setMetadata('category', div.find_next_siblings('div')[0].span.next_sibling.strip())
-            elif 'Rating:' == stripHTML(div).strip()[:7]:
+            if 'Rating:' == stripHTML(div).strip()[:7]:
```

Comment:
The two `stripHTML(___).strip()` calls are redundant--`stripHTML()` already does that.